Scripts/Tests/ApiTests.cs

```diff
@@ -0,0 +1,321 @@
+using NUnit.Framework;
+using System;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace CharismaSDK.Tests
+{
+    [TestFixture]
+    public class ApiTests
+    {
+        #region Test Variables
+
+        // Pre-made story metadata, used to inform the tests
+        private const int TEST_CASE_TOKENID = 15452;
+        private const int TEST_CASE_VERSIONID = -1;
+        private const string TEST_CASE_APIKEY = "cff59833-71fc-42c4-9077-220d42aef65a";
```

Comment:
Ideally I wouldn't want to expose these in code, could we pass them into the test suite as environment variables somehow instead?

Reply:
I had a look into it, and it doesnt seem unity supports that - the values need to be predefined in code via attributes. we could have an external file that loads them in and assigns em to the testdata however. I've added a comment for that on the latest update, but I think it may be something lower priority right now. 
Another thing to mention, the way I set up the unit tests, not every created Playthrough would pass all tests (for example, ForkPlaythrough requires you to have a published playthrough which may not be the case for all), which seems misleading.